main/lsp/requests/sorbet_read_file.cc

```diff
@@ -0,0 +1,25 @@
+#include "main/lsp/requests/sorbet_read_file.h"
+
+using namespace std;
+
+namespace sorbet::realmain::lsp {
+
+SorbetReadFileTask::SorbetReadFileTask(const LSPConfiguration &config, MessageId id,
```

Comment:
This used to be in `request_dispach.cc` as a lambda. I moved it into its own `LSPRequestTask`.